src/controls/dynamicForm/dynamicField/IDynamicFieldProps.ts

```diff
@@ -92,4 +93,30 @@ export interface IDynamicFieldProps {
   itemsQueryCountLimit?: number;
   customIcon?: string;
   orderBy?: string;
+  /** Used for customize component styling */
+  styles?:IStyleFunctionOrObject<IDynamicFieldStyleProps, IDynamicFieldStyles>;
+}
+
+
+export interface IDynamicFieldStyleProps {
+  theme: Theme; 
+  required?: boolean;
+}
+
+export interface IDynamicFieldStyles {
+ titleContainer: IStyle;
+ fieldIcon:IStyle;
+ fieldDisplay:IStyle;
+ fieldDisplayNoPadding:IStyle;
+ fieldContainer:IStyle;
+ fieldDescription:IStyle,
+ fieldLabel:IStyle;
+ labelContainer:IStyle;
+ pickersContainer:IStyle;
+ FieldEditor:IStyle;
```

Comment:
Shouldn't it be written like this?

```suggestion
 fieldEditor:IStyle;
```

Reply:
I have corrected the class name and related document (Actually I copied this name from old _DynamicForm.module.scss_ 
 file...)